pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -137,6 +139,22 @@ class HdRprApiImpl {
         //InitIfNeeded();
     }
 
+	rpr::FrameBuffer* GetColorFramebuffer() {
+		auto aovIter = m_boundAovs.find(TfToken("color"));
```

Comment:
Use existing `color` token instead. `HdAovTokens->color` for example

Reply:
Done